src/MoBi.UI/Services/MoBiApplication.cs

```diff
@@ -11,10 +11,11 @@
 
 namespace MoBi.UI.Services
 {
-   public class MoBiApplication : WindowsFormsApplicationBase
+   public class MoBiApplication : WindowsFormsApplicationBase, IMessageFilter
```

Comment:
where does this come from?

Reply:
I learned about IMessageFilter from a DevExpress forum discussion